rules.txt

```diff
@@ -1,12 +1,395 @@
-Submission Rules:
-  All submissions must NOT be:
-    Questions / requests for support.
-    NSFW.
-    Memes.
-    Links to sites that require logins or have a paywall.
-  All submissions MUST be:
-    The original article - no link shorteners OR spamblogs, please!
-    Compliant with the site-wide reddit rules.
-Comment Rules:
-  All comments MUST be:
-    Compliant with the site-wide reddit rules.
+#Submission Rules:
+#  All submissions must NOT be:
+#    Questions / requests for support.
+#    NSFW.
+#    Memes.
+#    Links to sites that require logins or have a paywall.
+#  All submissions MUST be:
+#    The original article - no link shorteners OR spamblogs, please!
+#    Compliant with the site-wide reddit rules.
+#Comment Rules:
+#  All comments MUST be:
+#    Compliant with the site-wide reddit rules.
```

Comment:
Are these still needed in rules.txt? Can we remove and leave README.md as the single definitive place for user-readable rules?